setup.sh

```diff
@@ -0,0 +1,24 @@
+#! /usr/bin/env bash
+
+OLD_PWD=$(pwd)
+
+# Additional ZShell configs
+ADDITIONAL_CONFIGS='source ~/.config/zshrc/*'
+if [[ ! $( grep "$ADDITIONAL_CONFIGS" ~/.zshrc ) ]] ; then
+  echo "Adding custom additional configs from $ADDITIONAL_CONFIGS to ~/.zshrc" ;
+  echo $ADDITIONAL_CONFIGS >> ~/.zshrc ;
+fi ;
+
+# Conda-zsh-completion
+echo 'Installing / updating conda-zsh-completion'
+git clone https://github.com/esc/conda-zsh-completion.git ~/.conda-zsh-completion
```

Comment:
This will clone master, which may not be stable. Probably better to find a way to get the latest release tag and then install from that.